app/src/main/java/emu/skyline/EmulationActivity.kt

```diff
@@ -263,7 +273,7 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
             }
         }
 
-        @Suppress("DEPRECATION") val display = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) display!! else windowManager.defaultDisplay
+        @Suppress("DEPRECATION") val display = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) display else windowManager.defaultDisplay
```

Comment:
Same issue as the `ContextCompat` case above, if `Context.getDisplay()` is null, we have bigger problems than the NPE crash.